postgres-debezium-ksql-elasticsearch/postgres-debezium-ksql-elasticsearch-docker-short.adoc

```diff
@@ -85,8 +85,8 @@ docker-compose_connect-debezium_1   /docker-entrypoint.sh start      Up      0.0
 docker-compose_datagen-ratings_1    bash -c echo Waiting for K ...   Up
 docker-compose_kafka-connect-cp_1   /etc/confluent/docker/run        Up      0.0.0.0:18083->18083/tcp, 8083/tcp, 9092/tcp
 docker-compose_kafka_1              /etc/confluent/docker/run        Up      0.0.0.0:9092->9092/tcp
-docker-compose_ksql-cli_1           sleep infinity                   Up
-docker-compose_ksql-server_1        bash -c echo Waiting for K ...   Up
+docker-compose_ksql-cli_1           /bin/sh                          Up
+docker-compose_ksql- server_1       /etc/confluent/docker/run        Up      8088/tcp
```

Comment:
is that space ` ` between dash and `server` correct?

Reply:
Ack and fixed, good catch